### ISSUE TITLE
lambda doesn't work,but %lambda does work.

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#760]
Imported from SourceForge on 2024-07-03 10:30:25
Created by **[nobody](https://sourceforge.net/u/nobody/)** on 2014-09-06 04:14:02
Original: https://sourceforge.net/p/maxima/bugs/2812

---

To get eigenvalues of an intricate 2x2 matrix,
My handmade expression can yield eigenvalues properly.
But,when I try the command
eigenvectors(X) ;
,error messages come out as follows.
(Sorry,garbage characters instead of Greek characters have appeared.)

~~~~
(%i1)  X:matrix([1,1],[1,1]) ;

(%o1) (
1 1
1 1
)
(%i2)  X[1,1]: alpha[1]*c[1]^2/3 + c[3]^3 - log(delta[1]) + f^5 ;

(%o2) f^5-log (?_1)+c_3^3+(?_1*c_1^2)/3
(%i3)  X[1,2]: exp( eta[1] + b*eta[2] ) - log(d) ;

(%o3) e^(?_2*b+?_1)-log (d)
(%i4)  X[2,1]: log( eta[1] + eta[2]/b ) + exp(d) ;

(%o4) e^d+log ((?_2)/b+?_1)
(%i5)  X[2,2]: log(alpha[2]*c[2]^5/5 - c[1]/5) + exp(delta[2]) - lambda[2] ;

(%o5) log ((?_2*c_2^5)/5-(c_1)/5)-?_2+e^(?_2)
(%i6)  X ;

(%o6) (
f^5-log (?_1)+c_3^3+(?_1*c_1^2)/3 e^(?_2*b+?_1)-log (d)
e^d+log ((?_2)/b+?_1)             log ((?_2*c_2^5)/5-(c_1)/5)-?_2+e^(?_2)
)
(%i8)  eigenvalue[1]: (X[1,1]+X[2,2])/2 + sqrt( (X[1,1]-X[2,2])^2  
+4*X[1,2]*X[2,1] )/2 ;

(%o8) (sqrt((f^5-log ((?_2*c_2^5)/5-(c_1)/5)-log  
(?_1)+c_3^3+?_2+(?_1*c_1^2)/3-e^(?_2))^2+4*(e^d+log  
((?_2)/b+?_1))*(e^(?_2*b+?_1)-log (d))))/2+(f^5+log  
((?_2*c_2^5)/5-(c_1)/5)-log (?_1)+c_3^3-?_2+(?_1*c_1^2)/3+e^(?_2))/2
(%i9)  eigenvalue[2]: (X[1,1]+X[2,2])/2 - sqrt( (X[1,1]-X[2,2])^2  
+4*X[1,2]*X[2,1] )/2 ;

(%o9) (f^5+log ((?_2*c_2^5)/5-(c_1)/5)-log  
(?_1)+c_3^3-?_2+(?_1*c_1^2)/3+e^(?_2))/2-(sqrt((f^5-log  
((?_2*c_2^5)/5-(c_1)/5)-log  
(?_1)+c_3^3+?_2+(?_1*c_1^2)/3-e^(?_2))^2+4*(e^d+log  
((?_2)/b+?_1))*(e^(?_2*b+?_1)-log (d))))/2
(%i10)  eigenvectors(X) ;

Maxima encountered a Lisp error:

  Error in PROGN [or a callee]: Caught fatal error [memory may be damaged]

Automatically continuing.
To enable the Lisp debugger set *debugger-hook* to nil.
~~~~

To remedy this,
only by changing "lambda" into "%lambda"
the command eigenvectors work properly either on wxMaxima or on TeXmacs.

Also reported at
http://ehc.ac/p/maxima/mailman/message/32458023/

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 10:30:27
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2014-09-08 23:57:57
Original: https://sourceforge.net/p/maxima/bugs/2812/#3197

---

- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -5,6 +5,7 @@
 ,error messages come out as follows.
 (Sorry,garbage characters instead of Greek characters have appeared.)
 
+~~~~
 (%i1)  X:matrix([1,1],[1,1]) ;
 
 (%o1) (
@@ -52,7 +53,7 @@
 
 Automatically continuing.
 To enable the Lisp debugger set *debugger-hook* to nil.
-
+~~~~
 
 To remedy this,
 only by changing "lambda" into "%lambda"

~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 10:30:30
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2014-09-08 23:57:58
Original: https://sourceforge.net/p/maxima/bugs/2812/#96c6

---

Enclosed code in four tildes for appropriate formatting.

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 10:30:34
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2014-09-10 19:30:55
Original: https://sourceforge.net/p/maxima/bugs/2812/#1c32

---

- **labels**:  --> freeof, lambda
- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-03 10:30:38
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2014-09-10 19:30:55
Original: https://sourceforge.net/p/maxima/bugs/2812/#2798

---

Fixed by commit [1651c5c9]. Closing this report.
